aws/resource_aws_api_gateway_authorizer.go

```diff
@@ -109,7 +109,7 @@ func resourceAwsApiGatewayAuthorizerCreate(d *schema.ResourceData, meta interfac
 	if v, ok := d.GetOk("authorizer_credentials"); ok {
 		input.AuthorizerCredentials = aws.String(v.(string))
 	}
-	if v, ok := d.GetOk("authorizer_result_ttl_in_seconds"); ok {
+	if v, ok := d.GetOkExists("authorizer_result_ttl_in_seconds"); ok {
```

Comment:
Nit: since this field has a default value to fall back on we could also directly use `d.Get()` and set the attribute directly during input creation @L96

Reply:
changed, it does work with just `d.Get()`